app/scripts/modules/core/src/application/config/customBanner/CustomBannerConfig.tsx

```diff
@@ -0,0 +1,231 @@
+            <thead>
+              <tr>
+                <th className="text-center">Enabled</th>
+                <th>Text</th>
+                <th className="custom-banner-config-color-option-column">Text Color</th>
+                <th className="custom-banner-config-color-option-column">Background</th>
+                <th />
+              </tr>
+            </thead>
+            <tbody>
+              {this.state.bannerConfigsEditing.map((banner, idx) => (
+                <tr key={idx} className="custom-banner-config-row">
+                  <td className="text-center">
+                    <input
+                      checked={banner.enabled}
+                      onChange={(e: React.ChangeEvent<HTMLInputElement>) => this.onEnabledChange(idx, e.target.checked)}
+                      type="checkbox"
+                    />
+                  </td>
+                  <td>
+                    <textarea
+                      className="form-control input-sm custom-banner-config-textarea"
+                      onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) => this.onTextChange(idx, e.target.value)}
+                      style={{
+                        backgroundColor: banner.backgroundColor,
+                        color: banner.textColor,
+                      }}
+                      value={banner.text}
+                    />
+                  </td>
+                  <td>
+                    <Select
+                      clearable={false}
+                      options={bannerTextColorOptions}
+                      onChange={(option: Option<string>) => this.onTextColorChange(idx, option)}
+                      optionRenderer={this.colorOptionRenderer}
+                      value={banner.textColor}
+                      valueRenderer={this.colorOptionRenderer}
+                    />
+                  </td>
+                  <td>
+                    <Select
+                      clearable={false}
+                      options={bannerBackgroundColorOptions}
+                      onChange={(option: Option<string>) => this.onBackgroundColorChange(idx, option)}
+                      optionRenderer={this.colorOptionRenderer}
+                      value={banner.backgroundColor}
+                      valueRenderer={this.colorOptionRenderer}
+                    />
+                  </td>
+                  <td>
+                    <button className="link custom-banner-config-remove" onClick={() => this.removeBanner(idx)}>
+                      <span className="glyphicon glyphicon-trash" />
+                    </button>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </div>
+        <div className="col-md-10 col-md-offset-1">
+          <button className="btn btn-block add-new" onClick={this.addBanner}>
+            <span className="glyphicon glyphicon-plus-sign" />
+            Add banner
```

Comment:
```suggestion
            {' '}Add banner
```
should put a space between the icon and the text

Reply:
Thanks Chris, just updated!